libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -303,15 +302,34 @@ void AP_MotorsHeli_Single::calculate_roll_pitch_collective_factors()
         _collectiveFactor[CH_1] = 1;
         _collectiveFactor[CH_2] = 1;
         _collectiveFactor[CH_3] = 1;
+    }
+
+    //Three-Servo H3-140 CCPM mixer factors
+    else if (_swash_type == AP_MOTORS_HELI_SINGLE_SWASH_H3_140) {
+        // aileron factors
+        _rollFactor[CH_1] = 1;
+        _rollFactor[CH_2] = -1;
+        _rollFactor[CH_3] = 0;
+
+        // elevator factors
+        _pitchFactor[CH_1] = 1;
+        _pitchFactor[CH_2] = 1;
+        _pitchFactor[CH_3] = -1;
 
-    }else{              //H1 Swashplate, keep servo outputs separated
+        // collective factors
+        _collectiveFactor[CH_1] = 1;
+        _collectiveFactor[CH_2] = 1;
+        _collectiveFactor[CH_3] = 1;
+    }
```

Comment:
same comment here as above.  This brace should be on the same line
} else {
add the comment after the brace like this
} else {  // H1 straight outputs, no mixing